### ISSUE TITLE
shallow net issue

### ISSUE THREAD

**Issue by SenHe** [imatge-upc/saliency-2016-cvpr#18]
Dear Pan,

 

I am Sen, a phd student from the university of Exeter. Recently I am doing the literature review of saliency prediction and found your work. Therefore, I want to reproduce your work using the code you provided on Github. But when I was training using your code, the training loss is still zero, and the validation loss is also very small. It seems that I was training a pretrained model. So is there any missing file on github for training your shallow network. And another question is the input dimension. What is the dimension of X, and y in train.py of the shallow file? Have you done any preprocessing for the input image and the ground truth? Thank you very much!
